src/main/kotlin/com/arabyte/arabyteapi/domain/auth/controller/AuthController.kt

```diff
@@ -0,0 +1,50 @@
+package com.arabyte.arabyteapi.domain.auth.controller
+
+import com.arabyte.arabyteapi.domain.auth.dto.AuthorizeResponse
+import com.arabyte.arabyteapi.domain.auth.dto.RegisterRequest
+import com.arabyte.arabyteapi.domain.auth.dto.RegisterResponse
+import com.arabyte.arabyteapi.domain.auth.dto.jwt.ReissueAccessTokenRequest
+import com.arabyte.arabyteapi.domain.auth.dto.jwt.ReissueAccessTokenResponse
+import com.arabyte.arabyteapi.domain.auth.service.AuthService
+import com.arabyte.arabyteapi.global.annotation.SwaggerCustomException
+import com.arabyte.arabyteapi.global.enums.CustomExceptionGroup
+import io.swagger.v3.oas.annotations.Operation
+import org.springframework.http.HttpHeaders
+import org.springframework.web.bind.annotation.*
+
+@RestController
+@RequestMapping("/auth")
+class AuthController(
+    private val authService: AuthService,
+) {
+    @Operation(
+        summary = "카카오 로그인", description = "카카오 AccessToken을 받아와 기존 회원의 경우 로그인 완료, 기존 회원이 아닌 경우 " +
+                "카카오 계정 정보를 DB에 저장만 합니다."
+    )
+    @PostMapping("/kakao/authorize")
+    @SwaggerCustomException(CustomExceptionGroup.KAKAO_LOGIN)
+    fun loginWithKakaoAccessToken(
+        @RequestHeader(HttpHeaders.AUTHORIZATION) authorizationHeader: String
+    ): AuthorizeResponse {
+        val kakaoAccessToken = authorizationHeader.removePrefix("Bearer ").trim()
+        return authService.authorizeWithKakao(kakaoAccessToken)
+    }
+
+    // TODO - jwt 처리 필요
+    @Operation(summary = "회원가입", description = "회원 가입이 완료되는 시점에 카카오 정보 기반으로 사용자를 DB에 저장합니다.")
```

Comment:
```suggestion
    @Operation(summary = "회원가입 정보 수정", description = "회원 가입 정보를 업데이트합니다. user 객체가 생성되지는 않습니다.")
```

회원가입에 대해서 이렇게 작성하는게 좀 더 혼선이 적지 않을까 싶네요

Reply:
사용자 입장에서는 회원가입 화면이 끝나야 회원가입하는 됐다고 생각하겠지만,, 사실 DB에는 웬만한 정보가 다 들어갔다는.. 
아무튼 개발하는 사람 입장에서는 위와 같이 수정하는게 나을 것 같네요! 반영하겠습니다.